cristin-import/src/test/resources/features/GeneralMappingRules.feature

```diff
@@ -211,6 +211,19 @@ Feature: Mappings that hold for all types of Cristin Results
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource has no projects
 
+  Scenario: The Cristin Result's HRCS values are used to generate the URIs for the NVA Resource
+    Given a valid Cristin Result
+    And the Cristin Result has the HRCS values:
+      | helsekategorikode | aktivitetskode |
+      | 4                 | 6.4            |
+      | 13                | 3.1            |
```

Comment:
3 digits means that there can be a 3 digit category. How about helsekategorikodes with 3 digits?